examples/react-fcl-demo/src/demo/SendTransaction.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {useState} from "react"
+import * as fcl from "@onflow/fcl"
+
+import Card from '../components/Card'
+import Header from '../components/Header'
+import Code from '../components/Code'
+
+const simpleTransaction = `\
+transaction {
+  execute {
+    log("A transaction happened")
+  }
+}
+`
+
+const SendTransaction = () => {
+  const [status, setStatus] = useState("Not started")
+  const [transaction, setTransaction] = useState(null)
+
+  const sendTransaction = async (event: any) => {
+    event.preventDefault()
+    
+    setStatus("Resolving...")
+
+    const blockResponse = await fcl.send([
+      fcl.getLatestBlock(),
+    ])
+
+    const block = await fcl.decode(blockResponse)
```

Comment:
FCL deals with this for you: https://github.com/onflow/flow-js-sdk/blob/master/packages/fcl/src/resolve/index.js#L14

Reply:
fixed